data-description/human-clinical-and-health-data.md

```diff
@@ -1,26 +1,145 @@
+
+### Ontologies
+An ontology is a formal description of knowledge as a set of concepts within a domain, and the relationships that exist between them. It ensures a common understanding of information and makes explicit domain assumptions, thus allowing organisations to make better sense of their data ([Fundamentals, O. (2022, December 16)](https://www.ontotext.com/knowledgehub/fundamentals/what-are-ontologies/#:~:text=An%20ontology%20is%20a%20formal,better%20sense%20of%20their%20data)). 
+
+An ontology, on its own, allows researchers to be specific about describing the phenomena they work with, such as in the annotations of datasets. This facilitates the identification and retrieval of relevant information. More information at the [RDMkit](https://rdmkit.elixir-europe.org/metadata_management#how-do-you-find-appropriate-vocabularies-or-ontologies).
+
+#### Considerations
+
+When developing and using health data ontologies, consider the following: 
+
+ * **Interoperability** is crucial to ensure that ontologies are compatible with various healthcare systems and data standards, such as HL7 and OMOP. This interoperability will facilitate data exchange and integration.
+ * **Scalability** is another key factor, as ontologies must be designed to accommodate the growing volume and complexity of health data, including new diseases and treatments. 
+ * **Usability** of ontologies is vital, ensuring they are user-friendly for healthcare providers and researchers, and supported by clear documentation and tools like the Ontology Lookup Service (OLS) and NCBO BioPortal.
+ * **Data quality** considerations involve ensuring that ontologies contribute to the accuracy, reliability, and completeness of health data. This is fundamental for high-quality data analysis and decision-making. 
+ * **Privacy and ethics** are paramount, addressing concerns related to data confidentiality and ethical use, which should be incorporated into the ontology design to guide responsible data handling practices.
+
+#### Existing Approaches
+
+ * For interoperability, standardisation is essential for maintaining consistency and accuracy in data representation, which can be achieved by utilising widely accepted vocabularies like ICD-10 and SNOMED CT.
+ * Tools like the Ontology Lookup Service (OLS) and NCBO BioPortal provide access to various biomedical ontologies, supporting standardisation efforts and improving data quality and utility. These resources help researchers and healthcare providers ensure that their data adheres to the highest standards of accuracy and consistency.
+    * [NCBO BioPortal](http://bioportal.bioontology.org/): NCBO BioPortal is an extensive repository and web-based platform that provides access to a wide array of biomedical ontologies and terminologies, enabling users to search, browse, and utilise these resources for research and data annotation purposes.
```

Comment:
Also in this section we might want to make use of our existing tools/add them to the tool list: https://github.com/martin-nc/infectious-diseases-toolkit/blob/martinc-hchd/_data/tool_and_resource_list.yml

Reply:
So I should add the Ontology Lookup Service (OLS) and NCBO BioPortal to the tool list (as they're not there)? I guess we don't add vocabularies like ICD-10?

I'll add links to HL7 and OMOP, since I see the links aren't in the text.

Reply:
Reason why I brought it up was because we do have them in the RDMkit added as a tool:

```
- description: EMBL-EBI's web portal for finding ontologies
  id: ontology-lookup-service
  name: Ontology Lookup Service
  registry:
    biotools: ols
    fairsharing: Mkl9RR
    tess: Ontology Lookup Service
  url: https://www.ebi.ac.uk/ols/index
- description: A comprehensive repository of biomedical ontologies
  id: bioportal
  name: BioPortal
  registry:
    biotools: bioportal
    fairsharing: 4m97ah
    tess: BioPortal
  url: https://bioportal.bioontology.org
- description: Collaborative effort to develob interoperable ontologies for the biological sciences
  id: the-open-biological-and-biomedical-ontology-foundry
  name: The Open Biological and Biomedical Ontology (OBO) Foundry
  registry:
    fairsharing-coll: bsg-d001083
  url: http://obofoundry.org
---

Reply:
vocabularies we don't do normally indeed!